mocking-testing/examples/src/code/personRepo.ts

```diff
@@ -0,0 +1,37 @@
+import { Person, PersonID } from './personModel';
+
+const cache = new Map<PersonID, Person>([
+  [
+    'person-1',
+    {
+      id: 'person-1',
+      name: 'John',
+      birthDate: new Date('1990-01-01'),
+    },
+  ],
+  [
+    'person-2',
+    {
+      id: 'person-2',
+      name: 'Jane',
+      birthDate: new Date('1995-01-01'),
+    },
+  ],
+]);
+
+export class PersonRepo {
+  static _is: number;
+
+  static get id(): number {
+    return PersonRepo._is;
+  }
+  static set id(value: number) {
+    PersonRepo._is = value;
+  }
```

Comment:
Odd that you named the property `_is` here instead of `_id`
```suggestion
export class PersonRepo {
  static _id: number;

  static get id(): number {
    return PersonRepo._id;
  }
  static set id(value: number) {
    PersonRepo._id = value;
  }
```

Reply:
ha, good catch.